### PR TITLE
CP - Add Hammer Katello 0.23.0 gem and bump package

### PR DESCRIPTION
**Pull request by chris1984**

### CODE REVIEW

**Comment by chris1984** [comment]
@jturel here is the cherrypick pr

File: packages/katello/rubygem-hammer_cli_katello/rubygem-hammer_cli_katello.spec [hunk 1]
```diff
@@ -96,6 +96,9 @@ install -m 0644 .%{gem_instdir}/config/%{plugin_name}.yml \
 %{gem_instdir}/test
 
 %changelog
+* Wed Sep 02 2020 Chris Roberts - 0.23-0.2.pre.master
```
Comment (jturel) [inline packages/katello/rubygem-hammer_cli_katello/rubygem-hammer_cli_katello.spec:99]: I think the prerelease stuff needs to be removed; check the 2.1 branch to see what was done there

**Comment by chris1984** [comment]
@jturel updated

**Comment by jturel** [comment]
LGTM!

**Comment by chris1984** [comment]
@tbrisker how does this look?

**Review by tbrisker — APPROVED** [review]
Thanks @chris1984 !

**Pull request merged**